RELEASE/scripts/sl_ascend.ash

```diff
@@ -14044,27 +14049,32 @@ boolean doTasks()
 
 	if(get_property("sl_beatenUpCount").to_int() > 5)
 	{
-		songboomSetting(3);
+		songboomSetting("dr");
+	}
+	else if ((get_property("sl_prewar") == "started") && (get_property("sl_war") != "finished"))
+	{
+		// Once we've started the war, we want to be able to micromanage songs
+		// for Gremlins and Nuns. Don't break this for them.
 	}
 	else
 	{
 		if((my_fullness() == 0) || (item_amount($item[Special Seasoning]) < 4))
```

Comment:
If I've understood this correctly, if a Vampyre reaches the Crypt with fullness 0, it would keep Songboom on `"food"`. However, `sl_ascend` seems to only eat after drinking, and only drink after reaching 1 adventure left. So if your first ~150 turns per day land in the Crypt, I *think* it's going to go through this condition.

It seems good to reserve eating until after trying to get Diet Pills... But I wonder if there's a way to tweak things so it might use the `"nightmare"` setting more often for the Crypt.

Reply:
What do you think of https://github.com/soolar/sl_ascend/pull/119/commits/49f29546ab6089c93653d1c9572c25b1db8f0138?

Reply:
Ack, I meant to add at the end that I'm really unfamiliar with how to use the Boombox. I bought one last week, but can't claim to understand the pros/cons of the different settings yet.

So on the one hand, that would collect more Nightmare Fuel 👍 On the other hand, I don't think I understand the consequences of losing Special Seasoning + food drops + spell damage from Sing Along.

I guess that's the benefit of having somebody else review the other PR, though :)

Reply:
Sure. Boombox does two things: a passive in each combat, and a consumable that drops after 11 combats. Switching songs doesn't change the consumable drop counter. This change says "we'll use food vibrations most of the time for the +food drop, but switch to nightmare intermittently for one fight to get the nightmare consumable instead of special seasoning."

The practical consequence is that we don't get any special seasoning until we complete the Cyrpt, which depending on leveling speed could cost as many turns as we consumed foodstuffs, but in exchange we get nightmare fuel, which seems good.

Reply:
Thanks for the explanation, makes sense now 🙇 So there's barely any loss in +food drop, that's good. And as long as Nightmare Fuel saves >1 turn, it's better than Seasoning.